docs/trino-connector/development.md

```diff
@@ -302,7 +306,9 @@ plugin.bundles=\
 node-scheduler.include-coordinator=true
 ```
 
-
+:::note
+Remove the file `/etc/catalogs/xxx.properties` if the corresponding `plugin/trino-xxx/pom.xml` is not recorded in the `/etc/config.properties`. For the hive plugin, please use  `plugin/trino-hive/pom.xml` after release version 435. Others should use `plugin/trino-hive-hadoop2/pom.xml`.
+:::
```

Comment:
What‘s the problem  about here?

Reply:
For example, if there exists `/etc/catalogs/blackhole.properties` but `/etc/config.properties` doesn't contain `plugin/trino-blackhole/pom.xml`, it will fail to start trino server with the following error msg
>ERROR main io.trino.server.Server No factory for connector 'blackhole'

And the hive plugin is another error of failing to start server, since the name changed in different trino version.